### ISSUE TITLE
amp-story documentation points to 1.0

### ISSUE THREAD

**Issue by newmuis** [ampproject/amphtml#14695]
The amp-story documentation at https://www.ampproject.org/docs/reference/components/amp-story.html refers to the 1.0 version of `amp-story` component instead of the 0.1 version.

The documentation should refer to 0.1 until 1.0 is ready.

**Comment by newmuis** [comment]
/to @bpaduch do you know what needs to be changed to fix this?  Is it just `amp-story.md`?

**Comment by bpaduch** [comment]
Yes, that's correct.  The [documentation on ampproject.org](https://www.ampproject.org/docs/reference/components/amp-story.html
) is imported from the [amp-story.md](https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/amp-story.md
) file. So, any changes made to amp-story.md will be reflected on ampproject.org (after being imported in the nightly production build).
